HACKING.md

```diff
@@ -0,0 +1,58 @@
+## Developing
+
+Use [Storybook](https://storybook.js.org/) to develop new components.
```

Comment:
Should this be a subheading?

### Developing components with Storybook

and then...

### Developing components with another project

Reply:
Done.